ipod/merge.py

```diff
@@ -187,6 +190,19 @@ def merge_and_extend_orbits(
             pc.is_in(search_summary_iter.orbit_id, orbits_iter.orbit_id)
         )
 
+        # if we are on at least the second iteration, accumulate the run_duration
+        if iterations > 0:
+            mask = pc.is_in(
+                search_summary_last_iter.orbit_id, search_summary_iter.orbit_id
+            )
+            search_summary_iter = search_summary_iter.set_column(
+                "run_duration",
+                pc.add(
+                    search_summary_last_iter.apply_mask(mask).run_duration,
+                    search_summary_iter.run_duration,
+                ),
```

Comment:
Does this mask guarantee sort order? I always forget. I suspect this might need to be masked, then sorted?

Reply:
Based on some simple testing (ex mask of alternating True, False), it appears to preserve sort order. Perhaps it isn't a guarantee though, maybe if the table is fragmented? I'll add a sort to be sure.